### PR TITLE
Rs 2023.05.24

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 1]
```diff
@@ -214,22 +222,23 @@ She told me that no matter what anyone around me thinks, I have to take that whi
...
 -- Page 50, Offset 8715.
 -- だから、見張り役の事でアルクェイドを憎むとか怨むとか、そういう気持ちはない。
-That's why I don't feel any resentment or malice toward Arcueid for making me her guard.
+That's why I don't feel any resentment or malice toward Arcueid for making me stand guard for her.
 }
 [sha:83a31c474a15eba7d9683f0445eb73e2f5550650]{
 -- Page 50, Offset 8716.
 -- せいぜい、厄介な事に巻きこまれてしまったな、と自分にぼやくぐらいだろう。
-At most, I would probably blame myself for getting caught up in this mess.
+If anything, I'd blame me for getting myself into this mess.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:230]: Misinterpretation

>At worst, I might grumble something about getting caught up in one hell of a mess, but that's about it.

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 2]
```diff
@@ -265,12 +274,14 @@ To begin with, the assumption that I would kill someone other than her is absurd
 [sha:508d74fece12fede5fe2b312e13faa17d851fcc3]{
 -- Page 55, Offset 8725.
 -- あんな気持ちになって、訳も判らないまま歩き回って、あげく殺してしまった相手なんて、今のところアルクェイドしかいないんだから。
-Getting such an urge, walking around aimlessly, and eventually killing someone... The only time that has ever happened was with Arcueid.
+Arcueid's the only one who's ever made me feel that way,
+ follow someone around without even thinking,
+ and finally kill them.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:279]: Something about this line is a bit strange

>Getting the urge to follow someone around for no reason only to end up killing them.
>For now, Arcueid is the only person who has made me feel that way.

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 3]
```diff
@@ -285,7 +296,8 @@ Rather, I don't think there's anyone other than her that I'd even want to――
 [sha:756c63eb6f9db157f1e5322d694bc176d364b2c0]{
 -- Page 56, Offset 8729.
 -- 「いや、忘れ物というか、なんていうか……考えてみれば、どうして俺は、おまえを殺そうなんて思ったんだろう……？」
-"No, it's not that I forgot something or anything like that... Come to think of it, why did I even want to kill you in the first place...?"
+"No, it's not like I %{i}forgot%{/i} something exactly, it's just...
+ why %{i}did%{/i} I want to kill you, anyway?"
 }
 [sha:97828db078bd1c1d426d5406edbf5944ce217037]{
 -- Page 57, Offset 8730.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:303]: 8731
>She has all the right to react like that.

>And why wouldn't she?

etc etc

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 4]
```diff
@@ -345,7 +357,7 @@ Then while I was still in shock, you halted my vital activity with your first at
 [sha:87c7b0e21a7ad6a6a282692c61f4e8e415461973]{
 -- Page 61, Offset 8741.
 -- 「うん、あの不意打ちは完璧だった。
-"Yeah, that surprise attack was absolutely flawless.
+"Yeah... that surprise attack was absolutely flawless.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:360]: In fact... your ambush was absolutely flawless.

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 5]
```diff
@@ -345,7 +357,7 @@ Then while I was still in shock, you halted my vital activity with your first at
...
 }
 [sha:01c7d5d0d6769fc11f45f891eff0bbfcc56a8b1b]{
 -- Page 61, Offset 8742.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:363]: >How flawless, you ask? Flawless enough that if you had captured that moment...

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 6]
```diff
@@ -521,22 +534,23 @@ It seems I've made her realize what she's put me through up to this point.
...
 -- Page 80, Offset 8778.
 -- 「……ふうん。それじゃあどうして、見ず知らずのわたしを殺したりしたのよ」
-"Hmm... If that's the case, why did you go and kill a complete stranger like me?"
+"Hmm... But then if that's the case, why'd you go and kill a complete stranger like me?"
 }
 [sha:5705ce51490808784717eb1bf8c3b6b3f10082a3]{
 -- Page 80, Offset 8779.
 -- 「―――だから、それが、」
-"―――Well, that's because..."
+"Well, that's because..."
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:553]: dash at end

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 7]
```diff
@@ -586,17 +600,17 @@ In this very room.
 [sha:29b024f5ed01a56aa195a3c00bf3bbc5b098c389]{
 -- Page 85, Offset 8789.
 -- 何の理由も、目的もなく、
-For no reason, with no purpose.
+For no reason. With no purpose.
 }
 [sha:d1b69dcd4b63c2c2b929f1d835dec6c12848f2b3]{
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:605]: her beautiful figure.

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 8]
```diff
@@ -611,12 +625,15 @@ Losing my ability to stand, I fall to my knees.
...
 [sha:1bdc2a17ef2fde1cd082ba81e41cc48a775cf18f]{
 -- Page 87, Offset 8795.
 -- たとえ相手が生きていて、それが人間じゃないとしても。
-Even though she survived. Even though she isn't human.
+// lint-off:LintEmDashes
+Even though she survived―――%{n}
+Even though she isn't human―――
 }
 [sha:d70a6935da9919cf8b4f3797f2b69a34e2caf0d6]{
 -- Page 87, Offset 8796.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:639]: probably use dash instead of colon

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 9]
```diff
@@ -1157,7 +1182,7 @@ I don't know whether it will have any effect on you, but I'll try to give you so
 [sha:6044fea21b4aaf93c5182d33398d3077e8761d46]{
 -- Page 133, Offset 8903.
 -- はい、と数枚の一万円札を差し出すアルクェイド。
-'Here', says Arcueid as she hands over a few 10,000 yen bills.
+Arcueid presents me with a handful of 10,000 yen bills.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:1185]: This right here is why Arcueid is the best GF in this game

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt [hunk 10]
```diff
@@ -1177,12 +1204,15 @@ Not that it really matters, but... did she just pull that stack of cash out of t
 [sha:97ce622bfc192ad730c77b3155d016f416958705]{
 -- Page 135, Offset 8907.
 -- 「無茶だけどやれ。眠ると体の機能は低下する。免疫力とか抵抗力な。
-"It's unreasonable, but do it anyway. Your bodily functions decrease when you go to sleep. Same goes for your immune system.
+"It's unreasonable, but do it anyway.
+ Your metabolism slows down when you sleep.
+ Same goes for your immune system.
```
Comment (Petrikow) [inline script/Arcueid/Day 5/05_00_ARC05_1B.txt:1209]: 無茶, one of the usual culprits

>%{i}Hmph%{/i}... Shiki, do you get how much your asking for here?"

>I know it's a lot, but do it anyway.

File: script/Arcueid/Day 5/05_00_ARC05_1B.txt (hunk 9 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 5/05_00_ARC05_1B.txt:1186]: Cosmic inflation, monetary inflation... she does it all

**Pull request merged**